models/DataObject/ClassDefinition/Service.php

```diff
@@ -282,6 +282,13 @@ public static function importObjectBrickFromJson($objectBrick, $json, $throwExce
     public static function generateLayoutTreeFromArray($array, $throwException = false, $insideLocalizedField = false)
     {
         if (is_array($array) && count($array) > 0) {
+            if ($array["name"] ?? null == "name") {
+                $name = $array["name"];
```

Comment:
```suggestion
            $name = $array['name'] ?? null;
            if ($name) {
```
Why should the name only checked if the value is "name"?